lib/X11/Xlib/Struct.pm

```diff
@@ -137,6 +137,12 @@ Access the scalar holding the bytes of the struct.
 sub bytes { ${$_[0]} }
 *buffer= *bytes;
 
+=head2 buffer
+
+C<buffer> is an alias tu L</C<bytes>>
+
+=cut
+
```

Comment:
I'm pretty sure what I was doing with the aliases was renaming a part of the API and providing backward-compatibility for any code that used 'buffer'.  I don't really want to advertise the old names anymore.  Same with 'id' vs. 'xid' below.